frontend/src/lib/lemon-ui/Popover/Popover.tsx

```diff
@@ -136,6 +136,10 @@ export const Popover = React.forwardRef<HTMLDivElement, PopoverProps>(function P
             ...(middleware ?? []),
         ],
     })
+
+    // We are just using this state to trigger a re-render, otherwise the autoUpdate won't always work
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    const [_, setFloatingRefElement] = useState<HTMLElement | null>(null)
```

Comment:
Tried to explain this in the comment but if the popup starts visible, it will never get the `autoUpdate` as setting the floating ref doesn't trigger a render